rasa/core/channels/channel.py

```diff
@@ -81,7 +81,9 @@ def __init__(
 def register(
     input_channels: List["InputChannel"], app: Sanic, route: Optional[Text]
 ) -> None:
-    async def handler(*args, **kwargs):
+    """Registers input channel blueprints with the application."""
```

Comment:
optional: `application` sounds a bit vague to me. `app` is Sanic server instance so I'd be tiny bit more explicit here
```suggestion
    """Registers input channel blueprints with Sanic."""
```